UI/PlayerListWnd.cpp

```diff
@@ -835,6 +862,8 @@ void PlayerListWnd::Refresh() {
     for (const auto& empire : Empires()) {
         int player_id = app->EmpirePlayerID(empire.first);
         auto player_row = GG::Wnd::Create<PlayerRow>(row_size.x, row_size.y, player_id, empire.first);
+        if (!clean_empire_status && (ready_empires.find(empire.first) != ready_empires.end()))
```

Comment:
There are other statuses... why is this one handled differently?

Reply:
There only two statuses and WAITING means empire ready with its orders and PLAYING_TURN means otherwise.

Reply:
Seems like the `PlayerRow` constructor or init should take or determine the status of players, and set itself internally when being constructed, rather than needing this separate set call outside...

Reply:
This value doesn't store anywhere except `PlayerRow`. Only if make it part of `Empire` class, but it would be a huge refactoring just before planned release.

Reply:
If `PlayerRow` can get the info for itself given the empire ID, then that's fine. Point is it's weird to have it as a separate call to set the status after...

Reply:
Status can be changed without refreshing players list.